aries-site/src/pages/foundation/color.mdx

```diff
@@ -0,0 +1,260 @@
+import { Button } from 'grommet';
+import { Figma } from 'grommet-icons';
+
+import { ColorSwatch } from '../../components';
+import {
+  ButtonRow
+} from '../../layouts';
+import { 
+  BackgroundDark,
+  BackgroundLight,
+  BordersDark,
+  BordersLight,
+  BrandColor, 
+  ContrastDark,
+  ContrastLight,
+  CorePalette, 
+  DarkModeContrast,
+  DarkPalette,
+  ElevationDark,
+  ElevationLight,
+  FocusColor,
+  GraphColors,
+  InputsDark,
+  InputsLight,
+  LightPalette,
+  LightModeContrast,
+  Overlay,
+  StatusDark,
+  StatusLight,
+  TextDark,
+  TextLight
+} from '../../examples';
+
+You will notice that the HPE Design System Color guidance is
+different from Brand Central. Prioritize the below HPE Design
+System color palette over Brand Central when identifying colors
+for your app or web based experience.
+
+<ButtonRow>
+  <Button
+    href="https://www.figma.com/file/eZYR3dtWdb9U90QvJ7p3T9/HPE-Color-Styles?node-id=590%3A1285"
+    icon={<Figma color="plain" />}
+    label="Open in Figma"
+    target="_blank"
+    secondary
+  />
+</ButtonRow>
+
+## Color Palettes
+
+Color is a key way we express our brand. Using color on
+interface elements, how typography is presented, displaying data
+visualizations, and adjusting context of the experience to light
+and dark are just some of the ways we illustrate inclusive
+design for our app and web based experiences.
+
+### Brand Color
+            
+HPE green symbolizes growth and vitality, commands attention and
+inspires action. Our green is an essential part of our brand
+identity and should be evident in every communication wherever
+possible. Always use the custom color specifications seen here
+to maintain consistency across channels and media.
+
+<BrandColor />
```

Comment:
So much cleaner to review the page with the example extracted : )

Reply:
totally